pallets/gear/src/lib.rs

```diff
@@ -1004,28 +1004,38 @@ pub mod pallet {
             let pages_config = PagesConfig {
                 max_pages: schedule.limits.memory_pages.into(),
                 lazy_pages_weights: LazyPagesWeights {
-                    signal_read: CostPerPage::new(schedule.memory_weights.lazy_pages_read),
-                    signal_write: CostPerPage::new(schedule.memory_weights.lazy_pages_write),
+                    signal_read: CostPerPage::new(
```

Comment:
can't we hide ref_time for some macro mb? Why the schedule should use exactly Weight type?

Reply:
seems carelessly for me